src/lib/i18n/LanguageKeys/Commands/XKCD.ts

```diff
@@ -0,0 +1,10 @@
+import { FT, T, Value } from '@skyra/http-framework-i18n';
+
+// Root
+export const RootName = T('commands/xkcd:name');
+export const RootDescription = T('commands/xkcd:description');
```

Comment:
Why not the format of https://github.com/skyra-project/archid-components/blob/main/packages/http-framework-i18n/src/lib/utils.ts#L131

Reply:
Because the keys are `name` and `description`, not `rootName` and `rootDescription`.